src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -1309,85 +1311,191 @@ public static unsafe int LastIndexOfAny(ref byte searchSpace, byte value0, byte
 
         // Optimized byte-based SequenceEquals. The "length" parameter for this one is declared a nuint rather than int as we also use it for types other than byte
         // where the length can exceed 2Gb once scaled by sizeof(T).
-        [MethodImpl(MethodImplOptions.AggressiveOptimization)]
```

Comment:
As it now has Sse2 intrinisics, removed the `AggressiveOptimization` which prevents them from being emitted at R2R.

Pure `Vector<T>` methods are blocked from R2R so need `AggressiveOptimization`  to bypass the inline restrictions at Tier0.

Note this is a regression on Arm as it will run Tier0 code; however I couldn't find a `#if` to put it behind; and it should get picked up by https://github.com/dotnet/runtime/issues/33308